modules/claims_api/app/clients/claims_api/bgs_client/definitions.rb

```diff
@@ -61,13 +61,47 @@ module ClaimantServiceBean
           )
       end
 
-      module ClaimantWebService
+      module ClaimantService
         DEFINITION =
           Service.new(
             bean: ClaimantServiceBean::DEFINITION,
             path: 'ClaimantWebService'
           )
 
+        module FindAssignedFlashes
+          DEFINITION =
+            Action.new(
+              service: ClaimantService::DEFINITION,
+              name: 'findAssignedFlashes',
+              key: 'return'
+            )
+        end
+
+        module AddFlash
+          DEFINITION =
+            Action.new(
+              service: ClaimantService::DEFINITION,
+              name: 'addFlash',
+              key: 'return'
+            )
+        end
+
+        module FindPoaByParticipantId
+          DEFINITION =
+            Action.new(
+              service: ClaimantService::DEFINITION,
+              name: 'findPOAByPtcpntId',
+              key: 'return'
+            )
+        end
+      end
+
+      module ClaimantWebService
+        DEFINITION =
+          Service.new(
+            bean: ClaimantServiceBean::DEFINITION,
+            path: 'ClaimantWebService'
+          )
         module FindPoaByParticipantId
           DEFINITION =
             Action.new(
```

Comment:
Iʼm still unclear why we have `ClaimantWebService` here (could not find that module/class anywhere) and why we have two `FindPoaByParticipantId`s (what is the difference between them?).

Reply:
I think I understand the `ClaimantWebService` module now. Should we consolidate the duplicate `FindPoaByParticipantId`s and move it, `FindAssignedFlashes`, and `AddFlash` under `ClaimantWebService`?